src/applications/vaos/components/TravelReimbursement.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getDaysRemainingToFileClaim } from '../utils/appointment';
+import {
+  selectAppointmentTravelClaim,
+  selectIsPast,
+  selectIsClinicVideo,
+  selectIsInPerson,
+} from '../appointment-list/redux/selectors';
+import { TRAVEL_CLAIM_MESSAGES } from '../utils/constants';
+import Section from './Section';
+
+export default function TravelReimbursement({ appointment }) {
+  const isPastAppointment = selectIsPast(appointment);
+  if (!isPastAppointment) {
+    return null;
+  }
+
+  const isInPerson = selectIsInPerson(appointment);
+  const isClinicVideo = selectIsClinicVideo(appointment);
+  // If it's not an in-person appointment or a clinic video appointment, don't show the link to file a claim
+  if (!isInPerson && !isClinicVideo) return null;
```

Comment:
nit: the travel pay eligibility logic looks the same between AppointmentTasks and TravelReimbursement. If this will always be the same, we should probably move the logic into the selector.js file. No changes needed here if this is expected to diverge in the future though.

Reply:
I moved all the shared logic to a new selector `selectIsEligibleForTravelClaim` I don't think the logic will diverge and if it does we can always make change it later.